api/Cargo.toml

```diff
@@ -14,6 +14,9 @@ mc-util-repr-bytes = { path = "../util/repr-bytes" }
 mc-util-serial = { path = "../util/serial" }
 mc-transaction-core = { path = "../transaction/core" }
 
+bs58 = "0.3.0"
+crc = "1.8.1"
+displaydoc = "0.1.7"
```

Comment:
We use `displaydoc = { version = "0.1", default-features = false }` everywhere else.

Reply:
I found `displaydoc = { version = "0.17", default-features = false }` in the lmdb crate

Reply:
Thats also good, its the default-features that should be added.